submit_and_compare/static/html/submit_and_compare_edit.html

```diff
@@ -18,7 +18,15 @@
           </div>
           <span class="tip setting-help">This assigns an integer value representing the weight of this problem</span>
         </li>
-        
+
+        <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+                <label class="label setting-label" for="submit_and_compare_edit_max_attempts">Max Attempts</label>
+                <input class="input setting-input" id="submit_and_compare_edit_max_attempts" step="1" min="1" value="{{ max_attempts }}" type="number">
```

Comment:
max_attempts is set to 0 (within the xblock) by default meaning that the user has unlimited attempts. The min is set to 1 in the the html for the purpose of supporting the buttons to click the up and down arrows to change the value. It turns out that setting the min to 0 does not prevent users from clicking the down arrow to get negative numbers. 1 on the other hand, enforces the bound. However because we initialize max attempts to 0, it shows up as 0 in the html by default. This is an irregularity with xblocks.